plugins/source/aws/resources/plugin/plugin.go

```diff
@@ -91,12 +91,11 @@ func titleTransformer(table *schema.Table) string {
 	return strings.Trim(strings.ReplaceAll(t, "  ", " "), " ")
 }
 
-func AWS() *source.Plugin {
-	return source.NewPlugin(
+func AWS() *plugin.Plugin {
+	return plugin.NewPlugin(
 		"aws",
 		Version,
-		tables(),
-		client.Configure,
-		source.WithTitleTransformer(titleTransformer),
+		New,
+		// plugin.WithTitleTransformer(titleTransformer),
```

Comment:
I think once we run doc generation some titles will be incorrect without this transformer, unless it was moved somewhere and I missed it

Reply:
Fixed